CHANGELOG.md

```diff
@@ -9,15 +9,15 @@ nav_order: 1
 
 ## [MAJOR.MINOR.PATCH] - YYYY-MM-DD
 
+## [3.6.1] - 2022-09-05
+
```

Comment:
please move all these changes under `[MAJOR.MINOR.PATCH] - YYYY-MM-DD` section
we add version number when we're ready to release